src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { shallow } from 'enzyme';
+import React, { Fragment } from 'react';
+
+import { Checkbox, CheckboxState } from './Checkbox';
+
+describe('<Checkbox />', () => {
```

Comment:
Voeg nog een test toe om te checken of:
- `id` correct wordt doorgegeven
- `label` correct wordt gerenderd